src/camera.rs

```diff
@@ -603,6 +642,15 @@ impl Camera {
 
         /// Recent Frame mode.
         mut recent_frame: XI_SWITCH::Type;
+
+        /// Setting the API debug level allows to select amount of messages stored to debug output.
+        mut debug_level: XI_DEBUG_LEVEL::Type;
+
+        /// Defines the size of the acquisition buffer in bytes. This is a circle buffer which contains image data from sensor.
+        mut acq_buffer_size: i32;
```

Comment:
```suggestion
        /// Removed
```
I agree that this functionality is missing, but unfortunately just adding the parameter is not a good solution. ACQ_BUFFER_SIZE is a 32-bit signed integer in the C API. The highest possible value would only get us to 2GB, which may be too small in a real-world application.
In the C API this is solved by introducing an additional parameter ACQ_BUFFER_SIZE_UNIT and if you really want to use this directly in Rust, you can do so using xiapi_sys. For these higher level bindings however I would suggest wrapping both of these parameters in a nicer function that also catches some of the potential errors (e.g. negative parameter values). 
I will deal with this in a separate ticket.

Reply:
#16 created for this.